### DISCUSSION TITLE
Hide Profile Settings for Specific User

### DISCUSSION THREAD

**Discussion by Lucas-SZ** [requarks/wiki#3233] [Help / Questions] [answered]
Hello Everyone,

Wiki.js 2.x -- Is it possible to hide the profile pages for a specific user?.  Use case is we have guest access disabled, and have a SAML request for a generic user titled "internal" through our internal systems set to view specific content.  We would like that user to not be able to make changes to the profile information (Change name, locale, etc..)

We have added a deny rule to read:assets and read:pages for /p and /p/profile but they do not appear to be honored.

Also, this was not an issue for our 1.x Wiki deployment.

Thanks 

![image](https://user-images.githubusercontent.com/73182423/111478598-9a1bd180-8706-11eb-9f17-5cb29f889fd1.png)

**Comment by NGPixel** [comment, marked as answer]
No, the profile page is always accessible to the user. Submit a feature request on https://feedback.js.wiki/wiki

**Reply by Lucas-SZ** [reply]
Thanks !
